quantum/encoder/tests/encoder_tests_split_no_right.cpp

```diff
@@ -100,11 +100,11 @@ TEST_F(EncoderSplitTestNoRight, TestOneClockwiseRightSent) {
     isLeftHand = false;
     encoder_init();
 
-    uint8_t slave_state[32] = {0xAA, 0xAA};
+    uint8_t slave_state[32] = {0};
     encoder_state_raw(slave_state);
 
-    EXPECT_EQ(slave_state[0], 0xAA);
-    EXPECT_EQ(slave_state[1], 0xAA);
+    EXPECT_EQ(slave_state[0], 0x0);
+    EXPECT_EQ(slave_state[1], 0x0);
```

Comment:
I suppose the intention here is to test that `slave_state` doesn't change during `encoder_state_raw` if there is no encoder on the opposite side. 

However since my change copies all encoder values over it would zero out these, as they are zero.